packages/backend-storage/src/access_builder.ts

```diff
@@ -9,37 +9,76 @@ import { StorageAccessBuilder } from './types.js';
 export const roleAccessBuilder: StorageAccessBuilder = {
   authenticated: {
     to: (actions) => ({
-      getResourceAccessAcceptor: getAuthRoleResourceAccessAcceptor,
+      getResourceAccessAcceptors: [getAuthRoleResourceAccessAcceptor],
```

Comment:
how many acceptors does `getAuthRoleResourceAccessAcceptor` return? 
If it only return 1, then why do we need to add `s` to the key name? 
If it returns more than 1, should we add `s` to it as well? Also, if it's more than one, would it be an array of acceptors inside `[]`?

I think I missed something here.

Reply:
Yeah this is a little weird now. Probably a "more accurate" name would be `getResourceAccessAcceptorGetters` but that seemed a little ridiculous...

To answer your question though the `getAuthRoleResourceAccessAcceptor` returns a single ResourceAccessAcceptor; the 's' was added to the end of the key name because it's now an array of getResourceAccessAcceptor functions.